<?xml version="1.0"?>
<launch>
    <arg name="fixed_frame_id" default="base_link"/>
    <arg name="frame_id" default="map"/>

    <node pkg="foresight" type="test_rrt.py" name="test_rrt"
        output="screen" />

    <node pkg="foresight" type="rrt_planner.py" name="rrt_planner"
        output="screen">
        <param name="frame_id" value="$(arg frame_id)" />
        <param name="fixed_frame_id" value="$(arg fixed_frame_id)" />
    </node>
        
    <node pkg="tf" type="static_transform_publisher" name="map_to_baselink"
        args="0 0 0 0 0 0 $(arg frame_id) $(arg fixed_frame_id) 100" />

</launch>
